internal/handlers/pg/msg_insert.go

```diff
@@ -62,34 +62,27 @@ func (h *Handler) MsgInsert(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
-	var inserted int32
+	ordered := true
+	if ordered, err = common.GetOptionalParam(document, "ordered", ordered); err != nil {
```

Comment:
Test for this is added but skipped (I'll cover it in the same issue but as a separate PR to not make this PR too big).